<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="ListaNeuronas">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>501</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonAAI">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar al inicio</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_voltaje">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>real</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_green">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>entero</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_blue">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>entero</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonAAF">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar al final</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonMostrar">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar neuronas</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_x">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>951</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_id">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>entero</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_red">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>entero</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_y">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>49</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>voltaje</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Posición en X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Posición en Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>red</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>green</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>49</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>blue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonGuardar">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>470</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonRecuperar">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>470</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Recuperar</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>ListaNeuronas</zorder>
   <zorder>ButtonAAI</zorder>
   <zorder>lineEdit_voltaje</zorder>
   <zorder>lineEdit_green</zorder>
   <zorder>lineEdit_blue</zorder>
   <zorder>ButtonAAF</zorder>
   <zorder>ButtonMostrar</zorder>
   <zorder>spinBox_x</zorder>
   <zorder>lineEdit_id</zorder>
   <zorder>lineEdit_red</zorder>
   <zorder>spinBox_y</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>ButtonGuardar</zorder>
   <zorder>ButtonRecuperar</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
